### PR TITLE
Frontend/resources

### PR DESCRIPTION
**Pull request by albertocruz6**
Resource page with examples and some functional links.

### CODE REVIEW

File: frontend/src/Routing/AppRoutes.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ import React from 'react'
...
 import Register from '../components/Register/Register'
+import Resources from '../components/Resources/Resources'
 
 export default function AppRoutes() {
     return (
         <BrowserRouter >
             <Routes>
                 <Route path='/' element={<Home />} />
-                <Route path="/register" element={<Register condition={false} />} />
+                <Route path="/register" element={<Register />} />
```
Comment (fabiolarobles1) [inline frontend/src/Routing/AppRoutes.js]: was the condition prop on register eliminated on purpose?

File: frontend/src/components/Resources/Resources.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+    margin: 60px auto;
+    min-height: 800px;
+}
+
+.resources-upper-sidescroll{
+    /* display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-evenly; */
```
Comment (fabiolarobles1) [inline frontend/src/components/Resources/Resources.css]: Do we need these commented lines on the code?

**Review by fabiolarobles1 — CHANGES_REQUESTED** [review]

File: frontend/src/Routing/AppRoutes.js (hunk 1 above)
Comment (albertocruz6) [reply frontend/src/Routing/AppRoutes.js]: It's below. Right under the /resources Route component in this file (Line 14). They only thing I changed was whether the condition was initially false and swapped it to true

File: frontend/src/components/Resources/Resources.css (hunk 2 above)
Comment (albertocruz6) [reply frontend/src/components/Resources/Resources.css]: No, will be deleting those.

File: frontend/src/Routing/AppRoutes.js (hunk 1 above)
Comment (albertocruz6) [reply frontend/src/Routing/AppRoutes.js]: Will revert changes in this area just in case.

**Review by fabiolarobles1 — APPROVED** [review]

**Pull request merged**